phpdotnet/phd/Package/PHP/Web.php

```diff
@@ -224,6 +224,28 @@ protected function writeJsonIndex() {
         $ids = array();
         $desc = array();
         foreach($this->indexes as $id => $index) {
+            // Skip indexes that are not chunks
+            if (!$index["chunk"]) {
+                continue;
+            }
+
+            // Handle items with long description but no short description
+            if (empty($index["sdesc"]) && !empty($index["ldesc"])) {
+                // Use long desc as short desc
+                $index["sdesc"] = $index["ldesc"];
+
+                // Find the parent book and use its long desc
+                $parentId = $index['parent_id'];
+                while (isset($this->indexes[$parentId])) {
```

Comment:
A code comment would make sense here to explain why `isset` is needed.

Reply:
Done